Makefile

```diff
@@ -14,33 +21,16 @@ help: ## - Show help message
 	@printf "${CMD_COLOR_ON} usage: make [target]\n\n${CMD_COLOR_OFF}"
 	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | sed -e "s/^Makefile://" | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 
-.PHONY: rpm
-rpm: ## - Build x86_64 linux RPM
-	@printf "${CMD_COLOR_ON} Build rpm\n${CMD_COLOR_OFF}"
-	@${DOCKER_BUILD} --ssh default  --target rpm -o ./  .
-
-.PHONY: image
-image:	## - Build the elastic fleet docker images
-	@printf "${CMD_COLOR_ON} Build the elastic fleet docker image\n${CMD_COLOR_OFF}"
-	${DOCKER_BUILD} --ssh default --target fleet -t fleet .
-
-.PHONY: run
-run: image ## - Run the smallest and secured golang docker image based on scratch
```

Comment:
nit: are we removing docker image build option?

Reply:
Yes, we don't need to build in docker. Its simple cross architecture building, which golang does natively.